src/admin-views/help-hub/navigation-links.php

```diff
@@ -24,5 +25,12 @@ class="tec-nav__tab <?php echo esc_attr( $hub_tab['class'] ); ?>"
 				<a class="tec-nav__link"><?php echo esc_html( $hub_tab['label'] ); ?></a>
 			</li>
 		<?php endforeach; ?>
+		<li
```

Comment:
Can we add an action here - then have the Controller add the link if TL is active?

Makes it easier to change later.